.drone.yml

```diff
@@ -0,0 +1,48 @@
+kind: pipeline
+name: amd
+
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+- name: build
+  image: ocaml/opam2:4.07
+  commands:
+  - sudo apt-get update && sudo apt-get -y install afl
+  - sudo chown -R opam .
+  - git -C /home/opam/opam-repository pull origin && opam update
+  - opam switch 4.07+afl
+  - opam pin add -n .
+  - opam depext cstruct-async cstruct-lwt cstruct-unix cstruct ppx_cstruct
+  - opam install -y .
+  - opam install -y crowbar fmt
+  - opam exec -- dune build fuzz/fuzz.exe
+  - mkdir out && mkdir in && echo bactrian > in/test
+  - opam pin add -y bun https://github.com/yomimono/ocaml-bun.git
+  - opam exec -- bun -i in -o out -- ./_build/default/fuzz/fuzz.exe
```

Comment:
Here's where I'd add a `timeout` invocation.  It's done via `dune runtest` in this version, but https://github.com/yomimono/ocaml-test-stdlib/blob/primary/test/jbuild#L29 has the basic shape of how this was done in earlier `bun` invocations.